apps/fpc/fsw/src/fpc_app.h

```diff
@@ -188,16 +182,6 @@ class FPC
     int8 m_Pos_Reset_Counter{0};				/* captures the number of times the estimator has reset the horizontal position*/
```

Comment:
The "{}" initializers will not be set correctly if the application is reloaded.  Move all these initializers to a function that gets called immediately at startup.

Reply:
#185